src/main/java/views/TeamEditorPane.java

```diff
@@ -0,0 +1,4 @@
+package views;
+
+public class TeamEditorPane {
+}
```

Comment:
Very nice and clear outline to read. The implementation looks very clean as well, good job and good luck on the rest!